federation/backends/stac_catalogs.md

```diff
@@ -0,0 +1,78 @@
+STAC in cases where it allows too much freedom.
+
+## Scope
+
+This profile aims to add a minimal set of requirements to STAC that is needed for a catalog to be compatible with openEO backends.
+Requirements that are optional, for instance because they enhance efficiency, are indicated as such.
+
+This profile is limited in scope to the most common types of raster EO datasets in the sense of how they are organized. It does not aim to define guidelines
+for very complex cases, or cover evey possible type of (EO) dataset. 
+
+### Dataset organization
+
+1. STAC collections will be mapped to openEO collections
+2. All STAC products in the same collection have the same set of 'bands', that are defined at collection level. This allows backends to look up the selected bands in a product.
+3. Assets in a product are raster files that contain a single band. 
+
+
+### Raster format
+
+1. Cloud native, georeferenced, raster formats shall be used. Common examples include (cloud optimized) geotiff and jpeg2000. 
+2. Rasters data is provided as regularly gridded data
+3. Georeferencing shall be based on standard projection systems.
+4. Overviews or pyramids are optional, they enhance viewing performance but are not required for processing at full resolution.
+
+### Data access
+
+1. STAC assets should contain a direct link to the raster file itself. 
+2. Assets links use the http(s) or S3 scheme.
+3. [HTTP RANGE](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) requests shall be supported, to request parts of an asset.
+
+### Authentication
+
+Integration of federated data access can be made more complex if the data provider imposes complex authentication schemes. 
+Currently we aim for catalogs that allow the openEO backend to log in with a single account for all data access. A more complex
+setup would be to make the login depend on the user making the openEO request.
+
+1. The STAC API itself should preferably be publicly accessible, avoiding the need for login.
+2. Asset links may require authentication.
+3. One of the following authentication schemes should be supported:
+3.1 Basic authentication
+3.2 S3 authentication headers
+
+## STAC metadata
+
+### Data cube extension
+https://github.com/stac-extensions/datacube
+
+
+### Projection information
+openEO backends require knowledge about the projection system, which should be provided at the collection level if it is the same for all products.
+Collections with multiple projections, for instance based on UTM, can specify it at product level.
+The [proj:epsg](https://github.com/stac-extensions/projection#item-properties-or-asset-fields) property is used to specify the projection.
+
+### Product geometry
+
+The product geometry specifies the footprint of the data in the raster file. It may be used by the backend for spatial filtering, 
+so any pixel outside of the footprint may not be loaded by the backend. The geometry is provided in EPSG:4326, but may be interpreted in the 
```

Comment:
This is invalid in STAC and contradicts with the "if this is perceived to be the case, the STAC specification should get precedence". The geometry must always be EPSG:4326, but you could provide a geometry and bbox in the native projection via `proj:geometry` and `proj:bbox`.

Reply:
But the geometry does specify the footprint in EPSG:4326 right? Or is the part about the interpretation wrong?
Maybe I should add that proj:geometry can be used as way to provide a less ambiguous geometry?

Reply:
Yes, the geometry must be EPSG:4326. proj:geometry can be used to specify a geometry in the native CRS (bbox works the same way). What I'm confused about is "but may be interpreted in the product projection system." - What does this mean/imply? I guess this should be dropped and replaced with a mention or the proj: fields?